bbin

```diff
@@ -543,10 +543,13 @@ WARNING: (We won't make any changes without asking you first.)
 
 (defn local-lib-path [script-deps]
   (let [lib (key (first script-deps))
-        coords (val (first script-deps))]
+        coords (val (first script-deps))
+        gitlibs-root (if-let [override (not-empty (System/getenv "GITLIBS"))]
+                       override
+                       "~/.gitlibs")]
```

Comment:
Hardcoding the "/" will maybe not works on Windows, which is probably why @rads had the other logic before. Can you revert that change?

Reply:
Sure. Does the tilde work on Windows?

Reply:
Just writing `(str (fs/path (fs/home) ".gitlibs"))` would probably the least magic.